sdk/storage/Azure.Storage.Files.DataLake/api/Azure.Storage.Files.DataLake.netstandard2.0.cs

```diff
@@ -295,7 +295,9 @@ public partial struct AccessControlChangeFailure
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
     public partial struct AccessControlChangeOptions
     {
+        private int _dummyPrimitive;
```

Comment:
What is this for?

Reply:
I'm actually not sure, it got added when I ran the Export_API script, but it's not an actual member variable of `AccessControlChangeOptions` but it appeared when I changed `bool?` to bool. But this also exists if you scroll down a bit to `AccessControlChangeResult`

Reply:
Is `AccessControlChangeResult` something we want to be public?

Reply:
yes

Reply:
Do we have anything like `_dummyPrimitive` anywhere else besides `AccessControlChangeOptions` and `AccessControlChangeResult`?

Reply:
We also have it existing in `AccessControlChangeCounters`, `AccessControlChangeFailure`, 'AccessControlChangeResult`, `AccessControlChanges`. It also exists in master `ReleasedObjectInfo`.

This also exists in the other packages in master, like for Blobs, like `AccessTier`, `BlobBlock`, `BlobErrorCode`, and `CustomerProvidedKey`

Reply:
Ok let's not worry about this.